pkg/applicationserver/io/packages/lora-cloud-device-management-v1/api/client.go

```diff
@@ -0,0 +1,109 @@
+// Copyright © 2019 The Things Network Foundation, The Things Industries B.V.
```

Comment:
We don't use dashes in package folders, except `./cmd` (but that's the exec name)

Reply:
Also, isn't the API v1? i.e. `pkg/applicationserver/io/packages/loradms/apiv1` ?

Reply:
I'd prefer to rename this to `pkg/applicationserver/io/packages/loradms/v1{/api}` since I didn't decouple the API from the package itself (with no v2 in sight in the near future I don't see a necessity in this, and the refactoring would be non-breaking).